src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -334,66 +395,98 @@ internal bool TryUnparse(StringBuilder sb)
             }
         }
 
-        public class ArgumentsCore
+        /// <summary>
+        /// The settings for <see cref="TextLoader"/>
+        /// </summary>
+        public class Options
         {
+            /// <summary>
+            /// Whether the input may include quoted values, which can contain separator characters, colons,
+            /// and distinguish empty values from missing values. When true, consecutive separators denote a
+            /// missing value and an empty value is denoted by \"\". When false, consecutive separators denote an empty value.
+            /// </summary>
             [Argument(ArgumentType.AtMostOnce,
                 HelpText =
                     "Whether the input may include quoted values, which can contain separator characters, colons," +
                     " and distinguish empty values from missing values. When true, consecutive separators denote a" +
                     " missing value and an empty value is denoted by \"\". When false, consecutive separators" +
                     " denote an empty value.",
                 ShortName = "quote")]
-            public bool AllowQuoting = DefaultArguments.AllowQuoting;
+            public bool AllowQuoting = Defaults.AllowQuoting;
 
+            /// <summary>
+            /// Whether the input may include sparse representations.
+            /// </summary>
             [Argument(ArgumentType.AtMostOnce, HelpText = "Whether the input may include sparse representations", ShortName = "sparse")]
-            public bool AllowSparse = DefaultArguments.AllowSparse;
+            public bool AllowSparse = Defaults.AllowSparse;
 
+            /// <summary>
+            /// Number of source columns in the text data. Default is that sparse rows contain their size information.
+            /// </summary>
             [Argument(ArgumentType.AtMostOnce,
                 HelpText = "Number of source columns in the text data. Default is that sparse rows contain their size information.",
                 ShortName = "size")]
             public int? InputSize;
 
             [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, HelpText = "Source column separator. Options: tab, space, comma, single character", ShortName = "sep")]
             // this is internal as it only serves the command line interface
-            internal string Separator = DefaultArguments.Separator.ToString();
+            internal string Separator = Defaults.Separator.ToString();
 
+            /// <summary>
+            /// The characters that should be used as separators column separator.
```

Comment:
> separators column separator [](start = 53, length = 28)

does it sound right for you?

Reply:
No, thank you for pointing out.

---
In reply to: [255807189](https://github.com/dotnet/machinelearning/pull/2417#discussion_r255807189) [](ancestors = 255807189)